great_expectations/checkpoint/checkpoint.py

```diff
@@ -690,8 +689,8 @@ def __init__(
         profilers: Optional[List[dict]] = None,
         validation_operator_name: Optional[str] = None,
         batches: Optional[List[dict]] = None,
-        ge_cloud_id: Optional[UUID] = None,
-        expectation_suite_ge_cloud_id: Optional[UUID] = None,
+        ge_cloud_id: Optional[str] = None,
+        expectation_suite_ge_cloud_id: Optional[str] = None,
```

Comment:
If these are being converted with a mm hook then shouldn't this be a union of `str | UUID` (and `None`)

Reply:
nope when we load a config object, the hook will activate and ensure UUID is not present.